docs/DOCS_README.md

```diff
@@ -2,39 +2,39 @@
 
 The documentation for Tendermint Core is hosted at:
 
-- <https://docs.tendermint.com/master/>
+- <https://docs.tendermint.com/main/>
```

Comment:
This is a broken link now. The link ending with `master` works.

But should we consider docs.tendermint.com without master or main at the end as the host of the main documentation?

Reply:
The link will work once we merge #9243.

<https://docs.tendermint.com/> automatically shows the latest **stable** docs (in our case, right now it shows the v0.34 docs).